form.js

```diff
@@ -1,26 +1,32 @@
-(function () {
-  "use strict";
-  const root = document.getElementById("root");
-  const form = document.createElement("form");
+const root = document.getElementById("root");
+const form = createNewEl('form');
 
+function createNewEl(el){
+  return document.createElement(el);
+}
 
-  const inputEmail = document.createElement("input");
-  inputEmail.type = "text";
-  inputEmail.placeholder = "Enter your email";
+function createNewAttr(el, atr, val){
+  return el.setAttribute(atr, val);
+}
 
+function appSubEl(parent, child){
```

Comment:
Don't reduce append to 'app'.
Get the second parameter as ...rest.
Append children's elements through the cycle.